### PR TITLE
Features/login

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by AbdullahMuhammed5 — COMMENTED** [review]
Great work, you just need to ignore the files under coverage folder.

**Comment by mohamedsgap** [comment]
Hey @mwissem please, take a look at `Coday` issue!

File: src/containers/Login/Login.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+import Checkbox from "@material-ui/core/Checkbox";
+import Link from "@material-ui/core/Link";
 import { Input } from "../../components/Input";
 import validateEmail from "../../utils/Validators";
 import "./Login.scss";
 
 function Login() {
-  const [isError, setisError] = useState(false);
-  const [errorMessage, seterrorMessage] = useState("");
+  const [isError, setisError] = React.useState(false);
```
Comment (aashrafh) [inline src/containers/Login/Login.js]: Why repeating ```React``` each time you use ```useState``` ? just import it once in the top of the file.

**Comment by ghost** [comment]
I fixed it now, Thanks.

**Review by aashrafh — APPROVED** [review]

**Review by mohamedsgap — APPROVED** [review]
LGTM :+1:

**Review by AbdullahMuhammed5 — APPROVED** [review]

**Pull request merged**